flujo/domain/dsl/step.py

```
@@ -1023,17 +1024,15 @@ def _exit_condition(output: Any, ctx: Optional[ContextModelT]) -> bool:
+    def use_input(self, key: str) -> "Pipeline[dict[str, object], StepOutT]":
         """Create a small adapter pipeline that selects a key from a dict input.
 
         This is a common pattern when working with :meth:`parallel` branches
         where each branch only needs a portion of the upstream output.
         """
 
-        async def _select(data: Any, *, context: BaseModel | None = None) -> Any:
-            if isinstance(data, dict):
-                return data.get(key)
-            raise TypeError("use_input expects a dict-like input")
+        async def _select(data: dict[str, object], *, context: BaseModel | None = None) -> StepInT:
```

Ruff (0.14.8)

Unused function argument: `context`

(ARG001)